app/models/alien.rb

```diff
@@ -1,2 +1,5 @@
 class Alien < ApplicationRecord
+    validates :name, presence: true
+    validates :distance, presence: true
+    validates :planet, presence: true
```

Comment:
you can put this in one line too :)
```suggestion
    validates :name, :distance, :planet, presence: true
```